app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
```

Comment:
The conf dates input is with a date picker, so that is covered. 
The problem I want to solve: it is currently possible to input an end date that is earlier than the start date. 
That is basically solved (I want to implement it differently than I did in this PR).
The next problem that came up: should the dates be `presence validate`d.  
TL;DR~My vote would be to leave out the presence validation. In theory orga's could enter a conf before the dates are set (RailsConf 2019).  I can imagine use cases for that. 

OTOH The data picker will set (both) the dates to today automatically on the client side, so why wouldn't we add the presence validation? It would always pass automatically, and It would make the chronology validation easier.  
Never heard orga's complain about the dates set upfront.~
:rubberduck: Conclusion: I add the presence validation.